src/foundry/client-esm/dice/terms/dice.d.mts

```diff
@@ -356,21 +346,28 @@ declare namespace DiceTerm {
   }
 
   interface KeepOrDropOptions {
-    keep?: boolean | undefined;
-    highest?: boolean | undefined;
+    /** Keep results?  (default: `true`) */
```

Comment:
When it's for a property (rather than a param) we use the `@defaultValue` JSDoc because it actually makes sense. Try:
```
/**
 * Keep results?
 * @defaultValue `true`
 */
```